### PR TITLE
Release note 1.26.0

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit is a release note for 1.26.0.

Related: #12299 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: docs/release/1.26/release-note-1.26.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Release Note 1.26.0
+
+## ONE Compiler
+
+- Support more Op(s): HardSwish, CumSum, BroadcastTo
+- Support more optimization option(s): `decompose_softmax`, `decompose_hardswish`, `FuseSliceWithTConv`,
```
Comment (jinevening) [inline docs/release/1.26/release-note-1.26.0.md:6]:
```suggestion
- Support more optimization option(s): `decompose_softmax`, `decompose_hardswish`, `fuse_slice_with_tconv`,
```

**Comment by jinevening** [comment]
For reviewers,

1.25 was released on Sep 06.

File: docs/release/1.26/release-note-1.26.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    `fuse_mul_with_conv`
```
Comment (jinevening) [inline docs/release/1.26/release-note-1.26.0.md:7]:
```suggestion
    `fuse_mul_with_conv`, `remove_unnecessary_add`, `fuse_horizontal_fc_layers`, `common_subexpression_elimination`,
    `remove_unnecessary_transpose`
```

File: docs/release/1.26/release-note-1.26.0.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- _one-quantize_ supports more option
+  - Requantization option that requantizes given model with different data types
+  - AMPQ option that finds high-quality mixed-precision configuration
+  - Saving min/max option
```
Comment (jinevening) [inline docs/release/1.26/release-note-1.26.0.md:11]:
```suggestion
  - Support a new option to save calibrated min/max values (--save_min_max)
```

File: docs/release/1.26/release-note-1.26.0.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Release Note 1.26.0
+
+## ONE Compiler
+
+- Support more Op(s): HardSwish, CumSum, BroadcastTo
+- Support more optimization option(s): `decompose_softmax`, `decompose_hardswish`, `FuseSliceWithTConv`,
+    `fuse_mul_with_conv`
+- _one-quantize_ supports more option
+  - Requantization option that requantizes given model with different data types
```
Comment (jinevening) [inline docs/release/1.26/release-note-1.26.0.md:9]:
```suggestion
  - Requantization option to convert TF2-quantized int8 model to uint8 model (--requantize)
```

File: docs/release/1.26/release-note-1.26.0.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  - AMPQ option that finds high-quality mixed-precision configuration
+  - Saving min/max option
+- Introduce _q-implant_ that writes quantization parameters and weights into the circle model
```
Comment (jinevening) [inline docs/release/1.26/release-note-1.26.0.md:12]:
```suggestion
- Add new parameters for moving average calibration (--moving_avg_batch, --moving_avg_const)
- Introduce _q-implant_ that writes quantization parameters and weights into the circle model
```

File: docs/release/1.26/release-note-1.26.0.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Release Note 1.26.0
+
+## ONE Compiler
+
+- Support more Op(s): HardSwish, CumSum, BroadcastTo
+- Support more optimization option(s): `decompose_softmax`, `decompose_hardswish`, `FuseSliceWithTConv`,
+    `fuse_mul_with_conv`
+- _one-quantize_ supports more option
+  - Requantization option that requantizes given model with different data types
+  - AMPQ option that finds high-quality mixed-precision configuration
```
Comment (jinevening) [inline docs/release/1.26/release-note-1.26.0.md:10]:
```suggestion
  - Add a new option to automatically find mixed-precision configuration (--ampq)
```

**Comment by mhs4670go** [comment]
This PR will be merged after changes of debian things. I'll post it after approving the contents of the release note.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**